portable/ThirdParty/GCC/Posix/port.c

```diff
@@ -103,6 +103,10 @@ static pthread_t hMainThread = ( pthread_t ) NULL;
 static volatile BaseType_t uxCriticalNesting;
 /*-----------------------------------------------------------*/
 
+#include <unistd.h>
```

Comment:
Can you consider moving the header to the top of the file, please. Here is our style guide: https://freertos.org/FreeRTOS-Coding-Standard-and-Style-Guide.html .

Reply:
moved